lib/monitor.js

```diff
@@ -214,3 +218,33 @@ internals.forOwn = (obj, func) => {
         func(key, obj[key]);
     }
 };
+
+internals.validateStream = (throwError, lastIndex, index, stream, reporterName, streamName) => {
+
+    if (stream instanceof Stream.Transform) {
```

Comment:
Is `instanceof` "good enough" here? I think this will break if someone does like "double inheritance". If they inherit from "good-console" for example... I'm not sure if `instanceof` will return `Stream.Transform`. It might not matter though... and is probably better than duck punching to check.